flink-contrib/flink-statebackend-rocksdb/src/main/java/org/apache/flink/contrib/streaming/state/RocksDBMapState.java

```diff
@@ -400,7 +410,7 @@ public UV setValue(UV value) {
 	/** An auxiliary utility to scan all entries under the given key. */
 	private abstract class RocksDBMapIterator<T> implements Iterator<T> {
 
-		static final int CACHE_SIZE_BASE = 1;
+		static final int CACHE_SIZE_BASE = 32;
```

Comment:
Why is 32 the magic value and not 16 or 64? Just asking :)

Reply:
And if memory is really a concern, then I would also think about releasing the raw bytes from the entry once we have lazily deserialized the objects from them.

Reply:
Indeed, I don't think the memory is really a concern (at least when CACHE_SIZE_LIMIT=128), I almost like to change the CACHE_SIZE to a fixed value (like 128), What do you think?